lib/chutney/TorNet.py

```diff
@@ -1327,11 +1334,13 @@ def getNodeDirInfoStatusPattern(self, dir_format):
            the ed25519 key. (Or RSA block matching, which is hard.)
         """
         nickname = self.getNick()
-        ed25519_id = None
-        # old tor versions do not have an ed25519-id
-        if 'ed25519_id' in self._env:
-            ed25519_id = self._env['ed25519_id']
-         # old tor versions do not have an ed25519-id
+        ed25519_key = self.getEd25519Id()
+         #old tor versions do not have an ed25519-id
```

Comment:
You can delete old code, rather than commenting it out.

Old code will always be there in the git history, if you need to go back to it.

Reply:
Sure, will be careful. Thank you